src/components/atom/IconButton/elements.ts

```diff
@@ -52,22 +38,25 @@ const floatIconButton = css`
   }
 `};`;
 
-export const IconButtonStyled = styled(Button)`
-    ${(props: IconButtonProps) => bMaincolor = props?.color ? props.color: "#FFFFFF00"
+export const IconButtonStyled = styled.button`
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    height: auto; 
+    display: flex;
+    align-items: center;
+    text-align: center;
+    background: ${(props: IconButtonProps) => bMaincolor = props?.color ? props.color: "#FFFFFF00"
```

Comment:
Please make use of color constants. Also apply to the rest of the code.

Reply:
just to confirm, are you referring to my use of the "#FFFFFF00" or the using of the props.color for adding the colors dynamically?